kernel/Dockerfile.kconfigx

```diff
@@ -0,0 +1,71 @@
+# syntax=docker/dockerfile:1.3-labs
+
+ARG BUILD_IMAGE
+
+FROM ${BUILD_IMAGE} AS kernel-build
+ARG KERNEL_VERSIONS
+ARG TARGETARCH
+
+RUN apk add \
+    argp-standalone \
+    bison \
+    build-base \
+    curl \
+    diffutils \
+    flex \
+    gmp-dev \
+    libarchive-tools \
+    mpc1-dev \
+    mpfr-dev \
+    ncurses-dev \
+    patch \
+    xz
+
+COPY / /
```

Comment:
The makefile sets `--no-cache` actually...

Reply:
Do you know why? It is also weird that it is doing a pull based on name, instead of for example doing a `--cache-from` from the previous image that would verify that cache is up-to-date and only pull if there is an actual cache match.